src/Commands/Authentication/LogoutCommand.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.Graph.Cli.Authentication;
+using Microsoft.Graph.Cli.Utils;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+
+namespace Microsoft.Graph.Cli.Commands.Authentication;
+
+class LogoutCommand
+{
+    private readonly LogoutService logoutService;
+
+    public LogoutCommand(LogoutService logoutService) {
+        this.logoutService = logoutService;
+    }
+
+    public Command Build() {
+        var logoutCommand = new Command("logout", "Logout by deleting the stored session used in commands");
```

Comment:
I'm curious, is there a logout option in Azure.Identity that sends a logout command to the identity endpoint so that the token can be invalidated? If so, we should be doing this in addition to deleting the stored session.

Reply:
Not as far as I can tell. The tokens are short lived though, so they expire after a short time.